Source/Service/Requests/EnrollUserPIN.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  EnrollUserPIN.swift
+//  Cotter
+//
+//  Created by Albert Purnama on 3/1/20.
+//
+
+import Foundation
+
+public struct EnrollUserPIN: APIRequest {
+    public typealias Response = CotterUser
+    
+    public var path: String {
+        return "/user/" + self.userID
+    }
```

Comment:
same, in addition: `"/user/\(self.userID)"`

^this feature is called string interpolation and generally we should prefer interpolation over concatenation in many diff languages, [here](https://teamtreehouse.com/community/what-is-the-difference-between-concatenation-and-interpolation-it-seems-like-you-can-do-the-exact-same-thing-with-both) is (a not so short reading) why. does not really apply here but just for future reference.